README.md

```diff
@@ -126,12 +126,11 @@ For more information, please read [http://docs.africastalking.com/airtime/sendin
   - `lastReceivedId`: "This is the id of the message that you last processed". Defaults to `0`
 
 
-- `createSubscription({ shortCode, keyword, phoneNumber, checkoutToken })`: Create a premium subscription
+- `createSubscription({ shortCode, keyword, phoneNumber })`: Create a premium subscription
 
   - `shortCode`: This is the premium short code mapped to your account. `REQUIRED`
   - `keyword`: A premium keyword under the above short code and mapped to your account. `REQUIRED`
   - `phoneNumber`:  The phone number to be subscribed. `REQUIRED`
-  - `checkoutToken`: This is a token used to validate the subscription request. `REQUIRED`
 
```

Comment:
@kevokevoh not sure why this still shows up on the diff; you'd made this commit yeah?